.gitignore

```diff
@@ -56,3 +56,5 @@ target/
 *~
 *.swp
 site/
+
+.idea
```

Comment:
When a project is opened with IDEA an .idea/ folder is automatically created. For many project types IDEA will build and sync its project files based on the project files in CM (eg, from maven, ivy, package.js, bundle, pretty much anything). So typically these files are not comitted. For this project there is really no reason at all to comit them and they should be considered the same as emacs '*~' files.